src/exoscale/vinyl/store.clj

```diff
@@ -45,7 +49,25 @@
     Protocolized so it can be called against the database or
     the store")
   (get-metadata [this]
-    "Return this context's record metadata"))
+    "Return this context's record metadata")
+  (new-runner [this]
+    "Return a runner to handle retryable logic"))
+
+(defn ^FDBDatabaseRunner runner-opts
```

Comment:
Sadly, `cond->` cannot be used here, all mutating functions on the runner return nil